tests/std/tests/P0753R2_manipulators_for_cpp_synchronized_buffered_ostream/test.cpp

```diff
@@ -56,9 +58,9 @@ void test_osyncstream_manipulators(string_buffer<typename Ty::char_type, ThrowOn
     assert(addressof(flush_emit(os)) == addressof(os));
     if constexpr (is_base_of_v<basic_osyncstream<char_type, traits_type, Alloc>, Ty>) {
         if constexpr (ThrowOnSync) {
-            assert(os.rdstate() == ios::badbit);
+            assert(os.rdstate() == ios_base::badbit);
         } else {
-            assert(os.rdstate() == (buf ? ios::goodbit : ios::badbit));
+            assert(os.rdstate() == (buf ? ios_base::goodbit : ios_base::badbit));

```

Comment:
I would prefer this ternary moved out of the assert
```suggestion
            if(buf) {
                assert(os.rdstate() == ios_base::goodbit);
            else {
                assert(os.rdstate() == ios_base::badbit);
            }
```